### ISSUE TITLE
Godot's Web Docs: WCAG 2.1 AA Guidelines

### ISSUE THREAD

**Issue by krs9851** [godotengine/godot-docs#8264]
**Your Godot version:**
Godot Engine 4.1 documentation in English 

**Issue description:**
The Godot Engine 4.1 web documentation is not WCAG (Web Content Accessibility Guidelines) 2.1 AA compliant. The WCAG 2.1 AA guidelines help users with disabilities access the web. If we do not follow them, we bar a portion of our user base from using the documentation. [An overview of why WCAG is important.](https://www.boia.org/blog/why-web-accessibility-is-important-4-reasons-to-create-accessible-content)

Some of the immediate issues are:
- Missing or incorrect alt text
- Links must have distinguishable text
- Links must be distinguishable without relying on color
- ARIA roles must have correct ARIA attributes

I plan to start work on this as a side project (I am a web accessibility specialist), and would love some extra help or advice for making it sustainable. 

**Steps to Check Your Page**
- Check the "alt" tags of all of your images. [How to write good alt text.](https://accessibility.huit.harvard.edu/describe-content-images) [How to add alt text to images in Sphinx.](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html)
- Check all of your links to make sure that they either have visible text, or an image with proper alt text. Make sure the text is descriptive of the link destination. 

I recommend checking for other accessibility issues that could be present with [the automated web accessibility checker, aXe by Deque](https://www.deque.com/axe/?branded=&utm_term=deque%20axe&utm_campaign=Search+-+axe+Pro+-+Branded&utm_source=adwords&utm_medium=ppc&hsa_src=g&hsa_ad=431265982275&hsa_tgt=kwd-926849200779&hsa_mt=p&hsa_ver=3&hsa_acc=7854167720&hsa_kw=deque%20axe&hsa_grp=76354481701&hsa_cam=6769485255&hsa_net=adwords&gclid=Cj0KCQjw4bipBhCyARIsAFsieCwidbhc-tBJCdWR7IenAfxxQPprXIDJHtpOtvaFHdZepKeD27NKrgwaArmVEALw_wcB), or by doing a manual check with your computer's built in assistive technology.
